render/src/main/java/org/codice/imaging/nitf/render/NitfRenderer.java

```diff
@@ -137,6 +148,61 @@ private void renderJPEG(final ImageSegment imageSegment, final Graphics2D target
         });
     }
 
+    private void renderJPEG2k(final ImageSegment imageSegment, final Graphics2D targetGraphic) throws IOException {
+        final ImageReader reader = getImageReader("image/jp2");
+        reader.setInput(imageSegment.getData(), true, true);
+        final ImageReadParam param = reader.getDefaultReadParam();
+
+        if (ImageRepresentation.MULTIBAND.equals(imageSegment.getImageRepresentation())) {
+            final int[] sourceBands = getSourceBands(imageSegment);
+            param.setSourceBands(sourceBands);
+        }
+
+        processBlocks(imageSegment, (r, c) -> {
+                    Rectangle rect = new Rectangle((int) (c * imageSegment.getNumberOfPixelsPerBlockVertical()),
+                            (int) (r * imageSegment.getNumberOfPixelsPerBlockHorizontal()),
+                            (int) imageSegment.getNumberOfPixelsPerBlockHorizontal(),
+                            (int) imageSegment.getNumberOfPixelsPerBlockVertical());
+                    param.setSourceRegion(rect);
+
+                    BufferedImage renderedBlock = reader.read(0, param);
+                    param.setDestination(renderedBlock);
+                    targetGraphic.drawImage(renderedBlock, (int) (c * imageSegment.getNumberOfPixelsPerBlockVertical()),
+                            (int) (r * imageSegment.getNumberOfPixelsPerBlockHorizontal()),
+                            null);
+
+                }
+        );
+    }
+
+    private int[] getSourceBands(final ImageSegment imageSegment) {
+        List<Integer> imageBands = new ArrayList<Integer>();
+
+        for (int i = 0; i < imageSegment.getNumBands(); i++) {
+            ImageBand band = imageSegment.getImageBandZeroBase(i);
+
+            if (null != band.getImageRepresentation()) {
```

Comment:
There are some YCbCr format images in the JITC samples. Do you want to try to handle this or just add a ticket for "Future"?